### ISSUE TITLE
Table Header Name Moved To aria-label Attribute

### ISSUE THREAD

**Issue by farroutt3fan** [mkoryak/floatThead#350]
Please note, this is not a bug, but an observation noted by our UI Automation team. I have not included a JSFiddle for it, because it can be seen on the main site's examples, and likely in all uses. 

The table header names are being removed from the <th> elements in the original table and placed inside the floated <th> table elements and also added as aria-label attributes on the original table. Our automation tries to look for them inside the <th> elements of the original table, but they are no longer there., They are able to work around the issue by finding the aria-label attribute,

So, I'm wondering if there is a reason this is done, and if it is not necessary, is it a behavior that can be changed in a future update to leave the heading names in the original table.

Thanks for a great library and for your attention!

**Comment by mkoryak** [comment]
This is how the plugin works. It does not use a clone of the header to make it float because it does not want to copy the events etc that may be bound to the header DOM. aria-label is for screen readers. 

I cannot add the header text into the original table because sometimes table headers contain html,  not just text, and adding html back  into the placeholder header could break the column alignment. 

**tldr:**
it was done for a good reason, and it will remain this way forever
